ledger/eval_test.go

```diff
@@ -1231,7 +1231,7 @@ func (eval *BlockEvaluator) txn(t testing.TB, txn *txntest.Txn) {
 	stxn := txn.SignedTxn()
 	err := eval.testTransaction(stxn, eval.state.child(1))
 	require.NoError(t, err)
-	eval.Transaction(stxn, transactions.ApplyData{})
+	err = eval.Transaction(stxn, transactions.ApplyData{})
```

Comment:
@jannotti FYI

Reply:
Thanks, I have that in my application-actions branch.